Gemfile

```diff
@@ -44,6 +44,7 @@ group :test, :development do
   gem 'pry-rails'
   gem 'rspec-rails'
   gem 'rack_session_access'
+  gem 'erb2haml'
```

Comment:
I don't think we actually want to commit this to the Gemfile, right? No need for it after the conversion.